tests/SmileIdentityCoreTest.php

```diff
@@ -343,11 +334,11 @@ public function testSubmitJobShouldRequiresAtLeastOneSelfieForJTOtherThanJT6():
         $this->sic->submit_job($this->partnerParams, $imageDetails, $this->idParams, $this->options);
     }
 
-    public function testGenerateKey(): void
+    public function testGenerateSignature(): void
     {
-        $timestamp = Clock::now()->getTimestamp();
-        $sec_key = $this->sic->generate_sec_key(false);
-        $this->assertEquals($timestamp, $sec_key["timestamp"]);
+        $timestamp = Clock::now()->format(DateTimeInterface::ISO8601);
```

Comment:
`DateTimeInterface::ISO8601` has been deprecated. can you update to `DateTimeInterface::ATOM`
```suggestion
        $timestamp = Clock::now()->format(DateTimeInterface::ATOM);
```